### PR TITLE
Remove some past free offerings of AWS and MS Azure

### PR DESCRIPTION
**Pull request by thispsj**
Removes some services which used to be Always Free but now 12 mo free or no longer free in AWS and MS Azure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,14 +94,10 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [CloudFront](https://aws.amazon.com/cloudfront/) - 1TB egress per month
     * [Cloudwatch](https://aws.amazon.com/cloudwatch/) - 10 custom metrics and 10 alarms
     * [CodeBuild](https://aws.amazon.com/codebuild/) - 100min of build time per month
-    * [CodeCommit](https://aws.amazon.com/codecommit/) - 5 active users,50GB storage and 10000 request per month
+    * [CodeCommit](https://aws.amazon.com/codecommit/) - 5 active users, 50GB storage and 10000 request per month
     * [CodePipeline](https://aws.amazon.com/codepipeline/) - 1 active pipeline per month
     * [DynamoDB](https://aws.amazon.com/dynamodb/) - 25GB NoSQL DB
-     * [EC2](https://aws.amazon.com/ec2/) - 750 hours per month of t2.micro or t3.micro(12mo). 100GB egress per month
```
Comment (ripienaar) [inline README.md:100]: This still have a free tier? As does EBS and RDS.

File: README.md [hunk 2]
```diff
@@ -113,8 +109,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [App Service](https://azure.microsoft.com/services/app-service/) - 10 web, mobile or API apps (60 CPU minutes / day)
     * [Functions](https://azure.microsoft.com/services/functions/) - 1 million requests per month
     * [DevTest Labs](https://azure.microsoft.com/services/devtest-lab/) - Enable fast, easy, and lean dev-test environments
-    * [Active Directory](https://azure.microsoft.com/services/active-directory/) - 500,000 objects
-    * [Active Directory B2C](https://azure.microsoft.com/services/active-directory/external-identities/b2c/) - 50,000 monthly stored users
```
Comment (ripienaar) [inline README.md:117]: Screenshot 2023-04-16 at 14 58 07

File: README.md (hunk 1 above)
Comment (thispsj) [reply README.md:100]: No all these services are now 12 month free not always free as per [this](https://aws.amazon.com/free/?sc_icampaign=acq_aws_takeover-1st-visit-free-tier&sc_ichannel=ha&sc_icontent=awssm-evergreen-1st-visit&sc_iplace=hero&trk=ha_awssm-evergreen-1st-visit&all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=tier%23always-free&awsf.Free%20Tier%20Categories=*all).
If you still believe this page is wrong then please let me know the right page as I am myself looking for this free tier information.

File: README.md (hunk 2 above)
Comment (thispsj) [reply README.md:117]: Yes the free tier is still there might have removed it by mistake. Sorry.

File: README.md (hunk 1 above)
Comment (ripienaar) [reply README.md:100]: The old one says "12mo" its always been 12 months and as per the top of the readme "at least a year" is the terms.

Comment (thispsj) [reply README.md:100]: Oh correct. I think I'll just close this. Thanks for the clarification.

**Pull request closed without merging**